lib/core/logs/log_level.dart

```diff
@@ -0,0 +1,72 @@
+import '../console/terminal_color.dart';
+
+/// A [logScope] is a way to group log messages by importance.
+enum LogLevel {
```

Comment:
How dare u criticize copy paste docs xD

Reply:
No, I actually liked this brief explanation. I don't know if I would have come up with something so short for something so 'trivial' ^^ You could have covered your tracks here though if you'd used `[LogLevel]` instead of `[logScope]` ;)

Oh, and in `log_scope.dart` you'll need to capitalize `[logScope]`